src/QsCompiler/TestTargets/Simulation/Target/Simulation.csproj

```diff
@@ -3,22 +3,15 @@
 
   <PropertyGroup>
     <AssemblyName>Simulation</AssemblyName>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
-    <OutputType>Exe</OutputType>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
I'm confused why this is being changed from 3.0 back to 2.1. Is this not getting .Net 3.0 anymore?

Reply:
The numbering is just a bit confusing: netcoreapp3.0 for an application corresponds to netstandard2.1 for a library (there is no netstandard3.0 - these two numbers are not kept in sync...)